### ISSUE TITLE
[BUG] Major League Baseball 2K10 -> Game freeze at the copyright screen

### ISSUE THREAD

**Issue by Glitcholog** [PCSX2/pcsx2#4486]
Describe the bug
As soon as the game boot, it will show the copyright screen, the game then stop there

To reproduce
Simply start the game , which will freeze in less than 30 seconds

Expected behavior
On a real Playstation 2, the game will boot and proceed as normal

GS Settings
Default GS Settings

Emulation Settings
Default settings and Software Mode

System Info (please complete the following information):

PCSX2 Revision: dev-1283
OS: Windows 10 (lastest update)
CPU: i7-8700K
GPU: NVIDIA GeForce RTX 2060 Super

Logs and Dumps

[majorleaguebaseball2k10.txt](https://github.com/PCSX2/pcsx2/files/6613246/majorleaguebaseball2k10.txt)

The last four row of the log are also shown here.

[VC IRX module][ VCMain ] IOP system memory Free:0x97400(619520) , MaxBlock:0x96700(616192)
padman: *** VBLANK OVERLAP ***
padman: *** VBLANK OVERLAP ***
	[VC IRX module][ VCMain ] IOP system memory Free:0x5f400(390144) , MaxBlock:0x5e700(386816)
padman: *** VBLANK OVERLAP ***

Other information

A personal dump of a Brand new (factory sealed) Major League Baseball 2K10

**Comment by prafullpcsx2** [comment]
You can download the patch file from here:-

https://forums.pcsx2.net/attachment.php?aid=73363

Just unzip it in your cheats folder and enable cheats.

**Comment by Glitcholog** [comment]
Thanks you!

Applied the patch and enabled cheat. This patch allowed me to get in the game itself!

I started a CPU VS CPU match in Play Ball mode and the game managed to go to end of the match, about 30 minutes later

Music and voice is all good, no noticeable glitches. Tried both default config and software mode.

![2k2k](https://user-images.githubusercontent.com/24578279/121212065-04966680-c84b-11eb-9c2c-a7843017a35f.PNG)

**Comment by atomic83GitHub** [comment]
Can you make sure all the ballparks works? I know this is long and seems dumb but I was lucky enough to catch a I-bit related bug in the "FLA - Pro Player Stadium". 

To properly test, you can enable this option in the console log and watch if MicroVU1 logs are consistently spammed during gameplay.

![image](https://user-images.githubusercontent.com/26351275/121219391-ef89f980-c883-11eb-98b8-3440fdf79786.png)

**Comment by Glitcholog** [comment]
Ill try to do so this week and ill report back (probaly in the weekend)

**Comment by Glitcholog** [comment]
I am sorry for the lack of response, sudden work priority tooks over.

Ive tried the following three arena , doing CPU matches, and so far, nothing seemed strange. Ill continue tonight with the rest of the arena.

**Comment by Glitcholog** [comment]
Ive let the game does a few more CPU vs CPU games, changing the team each time to get new arena, and so far, it seems good! Some of the field had rains in it, and the rains seemed to display correctly. I also did a few match on the PS2, especially in the rains one, and all was good.

**Comment by seta-san** [comment]
Isn’t this a dev9 issue?

**Comment by seta-san** [comment]
if you don't have a hdd connected the error in the log you are going to get is. "Unknown device 'hdd'". in this case i suspect that the emulator simply isn't reporting that there is no hdd properly

if you do have a harddrive connected the most likely error is. "DEV9: IF_CTRL Unknown Bit 3 Set"

it hits this bit of code in a condition that does nothing but reports that it's an unknown bit. it's probably supposed to do something. 
`if (value & (1 << 3))
				DevCon.WriteLn("DEV9: IF_CTRL Unknown Bit 3 Set");`

**Comment by refractionpcsx2** [comment]
Thanks for your valuable insight.

**Comment by F0bes** [comment]
> if you don't have a hdd connected the error in the log you are going to get is. "Unknown device 'hdd'". in this case i suspect that the emulator simply isn't reporting that there is no hdd properly
> 
> if you do have a harddrive connected the most likely error is. "DEV9: IF_CTRL Unknown Bit 3 Set"
> 
> it hits this bit of code in a condition that does nothing but reports that it's an unknown bit. it's probably supposed to do something. 
> `if (value & (1 << 3))
> 				DevCon.WriteLn("DEV9: IF_CTRL Unknown Bit 3 Set");`

Nope, that's normal. You can create devices from the IOP side with their respective read/write/etc handlers.
Because no HDD module is loaded (or it failed because there's no hard drive [normal behaviour] the `hdd` device is not created and bound.

**Comment by F0bes** [comment]
Disclaimer: My comment wasn't an explanation for the unknown bit being set (if I knew, it wouldn't be unknown 😏). It was for the message about no `hdd` device being available.
